kv/dist_sender.go

```diff
@@ -306,16 +306,16 @@ func (ds *DistSender) getFirstRangeDescriptor() (*proto.RangeDescriptor, error)
 	return &info, nil
 }
 
-// getRangeDescriptor retrieves the descriptor for the range
+// getRangeDescriptors retrieves the descriptor for the range
```

Comment:
the sentence needs an update now that you pluralized. Probably just remove everything and merge it with the second line.